runtime/src/iree/hal/drivers/vulkan/vulkan_device.cc

```diff
@@ -1488,6 +1488,26 @@ static iree_status_t iree_hal_vulkan_device_queue_execute(
       /*.signal_semaphores=*/signal_semaphore_list,
   };
   IREE_RETURN_IF_ERROR(queue->Submit(1, &batch));
+
+  if (command_buffer_count > 0) {
```

Comment:
this will need to be guarded on options->module like the other code we have (see L1520)